SEED-platform/seed
# 1978 mapquest api key error

#### Any background context you want to provide?
Alert user of geocoding error if no valid mapquest api key is saved.

#### What's this PR do?
Exit and print error message via geocode modal if no valid mapquest api key is detected in stead of hanging.

#### How should this be manually tested?
Under organization settings, input an invalid string for mapquest api key. Map and save your data:
<img width="361" alt="Screen Shot 2020-09-17 at 8 34 42 PM" src="https://user-images.githubusercontent.com/49968203/93551701-c5765d00-f92b-11ea-8028-5af8dcda60e1.png">

Error message is expected when a user tries to save the mapping:
<img width="655" alt="Screen Shot 2020-09-17 at 8 45 05 PM" src="https://user-images.githubusercontent.com/49968203/93551785-f6ef2880-f92b-11ea-93f7-cc92a7842a66.png">

Error message is also expected whe user tries to manually geocode selected properties/taxlots:
<img width="634" alt="Screen Shot 2020-09-17 at 9 16 48 PM" src="https://user-images.githubusercontent.com/49968203/93551846-22721300-f92c-11ea-82ff-3dc896e1a2ba.png">

After using a valid mapquest api key, one should expect to see success status:
<img width="689" alt="Screen Shot 2020-09-17 at 9 15 46 PM" src="https://user-images.githubusercontent.com/49968203/93551867-37e73d00-f92c-11ea-939c-b915163a5013.png">

<img width="637" alt="Screen Shot 2020-09-17 at 9 16 10 PM" src="https://user-images.githubusercontent.com/49968203/93551885-459cc280-f92c-11ea-83ca-cb7137d3f1a0.png">

#### What are the relevant tickets?
#### Screenshots (if appropriate)
